tests/testTriggers.go

```diff
@@ -10507,6 +10513,102 @@ func TriggerVolumeDriverDownVCluster(contexts *[]*scheduler.Context, recordChan
 
 }
 
+func TriggerSetDiscardMounts(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer ginkgo.GinkgoRecover()
+	defer endLongevityTest()
+	startLongevityTest(SetDiscardMounts)
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: SetDiscardMounts,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+
+	setMetrics(*event)
+
+	stepLog := "setting and unsetting discard mount options on regular intervals "
+	Step(stepLog, func() {
+		log.InfoD(stepLog)
+		log.Infof("Starting test case here !!")
+
+		for _, appNode := range node.GetStorageNodes() {
+			err := isNodeHealthy(appNode, event.Event.Type)
+			if err != nil {
+				UpdateOutcome(event, err)
+				continue
+			}
+			// Setting discard-mount-option on the node
+			err = SetUnSetDiscardMountRTOptions(&appNode, false)
+			if err != nil {
+				UpdateOutcome(event, err)
+				continue
+			} else {
+				// Verify if the cluster options set for run time parameters
+				optionsMap := []string{"NodeRuntimeOptions"}
+				cOptions, err := Inst().V.GetClusterOpts(appNode, optionsMap)
+				if !strings.Contains(cOptions["NodeRuntimeOptions"], "discard_mount_force:1") {
+					UpdateOutcome(event, err)
+					continue
+				}
+			}
+		}
+	})
+}
+
+func TriggerResetDiscardMounts(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer ginkgo.GinkgoRecover()
+	defer endLongevityTest()
+	startLongevityTest(ResetDiscardMounts)
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: ResetDiscardMounts,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+
+	setMetrics(*event)
+
+	stepLog := "setting and unsetting discard mount options on regular intervals "
+	Step(stepLog, func() {
+		log.InfoD(stepLog)
+		log.Infof("Starting test case here !!")
+
+		for _, appNode := range node.GetStorageNodes() {
+			err := isNodeHealthy(appNode, event.Event.Type)
+			if err != nil {
+				UpdateOutcome(event, err)
+				continue
+			}
+			// Setting discard-mount-option on the node
+			err = SetUnSetDiscardMountRTOptions(&appNode, true)
+			if err != nil {
+				UpdateOutcome(event, err)
+				continue
+			} else {
+				// Verify if the cluster options set for run time parameters
+				optionsMap := []string{"NodeRuntimeOptions"}
+				cOptions, err := Inst().V.GetClusterOpts(appNode, optionsMap)
+				if !strings.Contains(cOptions["NodeRuntimeOptions"], "discard_mount_force:0") {
```

Comment:
After setting and unsetting discard options , dont you need perfrom app validations?